distributions/aws/examples/rds-s3/README.md

```diff
@@ -0,0 +1,136 @@
+    - [yq](https://mikefarah.gitbook.io/yq) - A command line tool for YAML processing. (For Linux environments, use the [wget plain binary installation](https://mikefarah.gitbook.io/yq/#wget))
+    - [jq](https://stedolan.github.io/jq/download/) - A command line tool for processing JSON.
+    - [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) - A command line tool to customize Kubernetes objects through a kustomization file.
+
+2. Clone the repo and checkout the release branch
+
+```
+git clone https://github.com/awslabs/kubeflow-manifests.git
+cd kubeflow-manifests
+git checkout v1.4-branch
+```
+
+3. Create an EKS cluster 
+
+ Run this command to create an EKS cluster by changing `<YOUR_CLUSTER_NAME>` and `<YOUR_CLUSTER_REGION>` to your preferred settings. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
+
+```
+export CLUSTER_NAME=<YOUR_CLUSTER_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+
+eksctl create cluster \
+--name ${CLUSTER_NAME} \
+--version 1.19 \
+--region ${CLUSTER_REGION} \
+--nodegroup-name linux-nodes \
+--node-type m5.xlarge \
+--nodes 5 \
+--nodes-min 1 \
+--nodes-max 10 \
+--managed
+```
+
+4. Create S3 Bucket
+
+Run this command to create S3 bucket by changing `<YOUR_S3_BUCKET_NAME>` and `<YOUR_CLUSTER_REGION` to the preferred settings.
+
+```
+export S3_BUCKET=<YOUR_S3_BUCKET_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
+```
+
+5. Create RDS Instance
+
+Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. 
+
+
+### 2. Configure Kubeflow Pipelines
+
+1. Configure the following files in `distributions/aws/apps/pipelines` directory:
+
+    1. Configure `distributions/aws/apps/pipelines/params.env` file with the RDS endpoint URL, S3 bucket name, and S3 bucket region that were configured when following the steps in Create RDS Instance and Create S3 Bucket steps in prerequisites(#1-prerequisites). 
+        - For example, if your RDS endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, S3 bucket name is `kf-aws-demo-bucket`, and s3 bucket region is `us-west-2` your `params.env` file should look like:
+        - ```
+          dbHost=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+
+          bucketName=kf-aws-demo-bucket
+          minioServiceHost=s3.amazonaws.com
+          minioServiceRegion=us-west-2
+          ```
+    1. Configure `distributions/aws/apps/pipelines/secret.env` file with your RDS database username and password that were configured when following the steps in Create RDS Instance.
+        - For example, if your username is `admin` and your password is `Kubefl0w` then your `secret.env` file should look like:
+        - ```
+          username=admin
```

Comment:
Can you add an aestrik to change the password if user is using default CFN from step 5 in perquisite.

Reply:
done